docusaurus/docs/recipes/dropdown.md

```diff
@@ -0,0 +1,427 @@
+
+- `toggleMenu` - toggle the menu open state
+- `openMenu` - opens the menu
+- `reset` - resets downshift's state
+
+ℹ️ You can see the full list of _actions_ [here](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect#actions).
+
+#### 2. Component Usage
+
+Here is an example of how to use the `<Dropdown>` component we've built so far:
+
+```js
+import React, { useCallback } from 'react';
+import items from './items';
+import Dropdown from './Dropdown';
+
+export default function App() {
+  const handleStateChange = useCallback((state) => console.log(state), []);
+
+  return (
+    <div>
+      <h2>Dropdown with Downshift 🏎</h2>
+      <Dropdown items={ items } onStateChange={ handleStateChange } />
+    </div>
+  );
+}
+```
+
+#### 3. Native _select_ behavior support
+
+On mobile/touch devices the use of native _select_ behavior it's highly recommended due to UX reasons. To implement that, our `<Dropdown>` component is going to need some enhancements. Our approach will place a native select element on top of our trigger button, with the same items that we are passing to our `<Dropdown>` component, but it will invisible.   
+Whenever an option is selected, we need to update the internal state of `Downshift` so that those changes are reflected on the custom `<Dropdown>`.
+
+Take a look at the following steps:
+
+**1. Create a `<NativeSelect>` component**
+
+```js
+// NativeSelect.js
+
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+const NativeSelect = ({
+    items,
+    onChange,
+    placeholderLabel,
+    ...rest,
+}) => {
+    const placeholderOption = (
+        <option
+            disabled
+            value={ placeholderLabel }
+            key="native-option-disabled">
+            { placeholderLabel }
+        </option>
+    );
+
+    const selectableOptions = items.map((item, index) => (
+      <option key={ `native-option-${index + 1}` } value={ item.value }>
+        { item.label }
+      </option>
+    ));
+
+    return [placeholderOption, ...selectableOptions];
+    }, [items, placeholderLabel]);
+
+    const handleChange = (event) => {
+        const selectedItem = items.find((elem) => elem.value === event.target.value);
+        onChange(selectedItem);
+    };
+
+    return (
+        <select
+            { ...rest }
+            onChange={ handleChange }
+            defaultValue={ placeholderLabel }>
+            { renderOptions() }
+        </select>
+    );
+};
+
+NativeSelect.propTypes = {
+  items: PropTypes.array.isRequired,
+  placeholderLabel: PropTypes.string
+  onChange: PropTypes.func.isRequired,
+};
+
+export default NativeSelect;
+```
+
+⚠️ Please note we are passing a `placeholderLabel` so that the native select gets similar to the custom dropdown. You might not need it. You must be critical about what makes sense to your case scenario.
```

Comment:
```suggestion
⚠️ Please note we are passing a `placeholderLabel` so that the native select is similar to the custom dropdown. You might not need it. You must be critical about what makes sense to your case scenario.
```

I'm not sure if my corrections fits the meaning you wanted or if there's a missing word there